packages/components/navbar/examples/BasicNavbarExample.tsx

```diff
@@ -5,17 +5,7 @@ import { EntryIcon, ImageSquareIcon } from '@contentful/f36-icons';
 export default function BasicNavbarExample() {
   return (
     <Navbar
-      switcher={
-        <Navbar.Switcher logo="">
-          <Navbar.SwitcherItem isCircle>
-            <span role="img" aria-label="org logo">
-              ⚡️
-            </span>
-          </Navbar.SwitcherItem>
-          <Navbar.SwitcherItem>Space</Navbar.SwitcherItem>
-          <Navbar.SwitcherItem envVariant="master">master</Navbar.SwitcherItem>
-        </Navbar.Switcher>
-      }
+      switcher={<Navbar.Switcher envVariant="master">Space</Navbar.Switcher>}
```

Comment:
it's import Navbar from the '@contentful/f36-components' stable version, the envVariant prop doesn't exist there.
I think we may need to import directly from the navbar package

Reply:
I don't think we can do that in examples, I will revert examples.